### PR TITLE
Lodge MegaSeed Servitor

### PR DESCRIPTION
**Pull request by AysheDaArtist**
## About The Pull Request
<details>
<summary>
	Simple fix of adding a MegaSeed Servitor to Lodge's farm so they can actually use the medical plants specifically made for them.
<hr>
Once Xenobotany test PR is fully merged, this vendor can be also be stocked with the new plants while also keeping their original stock

</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
- Add: MegaSeed Servitor independent to Lodge
- Change: Replaces Lodge MegaSeed Servitor with it's own independent vendor seperate of Xenobotany

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
What's up with the map? it's failing linters.

File: code/game/machinery/vendor/food_vendors.dm [hunk 1]
```diff
@@ -255,6 +255,105 @@
...
+	auto_price = FALSE
+
+/**
+ * Populate hydroseeds product_records
+ *
+ * This needs to be customized to fetch the actual names of the seeds, otherwise
+ * the machine would simply list "packet of seeds" times 20
+ */
+
+/obj/machinery/vending/hydroseedslodge/build_inventory()
```
Comment (hex37) [inline code/game/machinery/vendor/food_vendors.dm:337]: This proc shouldn't be needed.

**Comment by Trilbyspaceclone** [comment]
You didnt use strongDMM thus the map was reformated in a way that is bad

**Comment by AysheDaArtist** [comment]
Don't usually make map edits

I'll re-do the PR with StrongDMM and remove build_inventory()

Thank you!

**Pull request closed without merging**